packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -2108,6 +2108,22 @@ void main() {
     ));
   });
 
+  testUsingContext('does not remove the test/ directory when recreating an application project with the --empty flag', () async {
+    await _createProject(
+      projectDir,
+      <String>['--no-pub', '--empty'],
+      <String>[],
+    );
+
+    projectDir.childDirectory('test').childFile('example_test.dart').createSync(recursive: true);
+
+    return _createProject(
+      projectDir,
+      <String>['--no-pub', '--empty'],
+      <String>['test/example_test.dart'],
+    );
+  });
```

Comment:
It looks like this test is missing an assertion

Reply:
`_createProject` implicitly asserts that `test/example_test.dart` exists but I agree with adding an explicit assertion too. 

Will include it.